chart/Chart.yaml

```diff
@@ -16,13 +16,13 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.0.0
+version: 2.0.0
```

Comment:
Has it been decided to match the helm chart versions to appVersions always? On one side, it's good that upgrade code will likely have one less thing to worry about.

Reply:
For now only for this next release to keep things simpler; initially we thought maybe we can leave the path version differ and pin major and minor, but that might have drawbacks, so we probably need to come up with a longer term strategy going forward - including who's suppose to increment the chart version, each PR that goes in, or a bot?